lib/docker.js

```diff
@@ -130,14 +130,26 @@ function createDockerFile() {
     }
 
     contents = 'FROM ' + baseImg + '\n' +
-        'RUN apt-get update && apt-get install -y ' + extraPkgs.join(' ') +
+        'RUN apt-get update --fix-missing && apt-get install -y ' + extraPkgs.join(' ') +
```

Comment:
Why? This shouldn't be needed, all of our container images start from a clean distro image.